biopandas/mmcif/pandas_mmcif.py

```diff
@@ -68,7 +67,7 @@ def read_mmcif(self, path):
         self.code = self.data["entry"]["id"][0].lower()
         return self
 
-    def fetch_mmcif(self, pdb_code: Optional[str] = None, uniprot_id: Optional[str] = None, source: str = "pdb"):
+    def fetch_mmcif(self, pdb_code: Optional[str] = None, uniprot_id: Optional[str] = None, source: str = "pdb", af2_version: int = 2):
```

Comment:
This would work, but i wonder if it wouldn't make more sense to make it part of the string, otherwise, we end up with a very specific argument that only makes sense if users select "alphafold2"?

Reply:
I think setting it in the string is still problematic as it would require an update to biopandas to support each new release of the database before it can be used

Reply:
Hm, I feel like this kind of complicated the user-interface too much. This is a value very specific to one particular "source" choice. Sure, we only have 2 source choices at the moment, but there could be more in the future (e.g., competitors to AF2, potentially AF3 etc.) and then it would become very messy for users to figure out which options go together. 

I feel like it would be much simpler to support a fixed number of options for the sake of user-friendliness. If there is a v3 in the future, I don't think it would be much effort to update.

Reply:
Sure, I've made the changes :)

Reply:
Nice, thanks!